### DISCUSSION TITLE
colab randomly shutting itself down?

### DISCUSSION THREAD

**Discussion by Headupdef** [TheLastBen/fast-stable-diffusion#2070] [Q&A]
is anyone else experiencing their colab shutting itself down? this has only been happening the past two days

**Comment by TheLastBen** [comment]
if it's free colab, then yes, it will kick you out often

**Comment by Headupdef** [comment]
Is this something they started doing recently? It never used to do it this
bad

**Reply by ClaraSanders** [reply]
I'm using Google Colab + and it also works like garbage. It is just Google being the amazing company we know and love. It performs so badly that last week I was unable to use Colab for a day because Drive didn't work so Colab never connected. Sometimes I try to upload a 1.99 gig model and the Drive give me the "estimated" upload time of 3000 hours.

**Reply by TheLastBen** [reply]
give runpod or paperspace a try, I have A1111 and dreambooth notebooks there

**Reply by ClaraSanders** [reply]
To be honest I'm a noob when it comes to this stuff. I just get into the AI world when a friend got me into Midjourney beta and my mind was blown away. I have no idea how runpod or paperspace works. I'll stick to Google while I send messages to Google customer attention telling them to "fix your shit" until I get tired of paying.

**Reply by TheLastBen** [reply]
using runpod or paperspace is fairly simple, I made the notebooks as close as possible to the colab version
https://github.com/TheLastBen/fast-stable-diffusion
scroll down and click on the thumbnails

**Comment by Headupdef** [comment]
Do u have a link to those notebooks

**Reply by TheLastBen** [reply]
https://github.com/TheLastBen/fast-stable-diffusion
scroll down and click on the thumbnails

**Reply by Rasad0** [reply]
bro can you do something with the colab code instead so it doesn't terminate the execution in the free version pls :')

**Comment by dilectiogames** [comment]
it doesn't shut down because of lack of resources. they tell you when they do that. you can get back in immediately and still gives you the GPU

I wanted to say it happens to me really often recently. is like you get onto a bad slot in the google server. once you find one slot that works it will run for hours perfectly.

it takes a couple of times till it works.

**Comment by Headupdef** [comment]
Mine keeps ctrl-c and shutting down or comes up with a ton of errors which
i assume is it having a problem connecting to Google drive

**Reply by dilectiogames** [reply]
i never been able to fix a notebook once it keeps throwing errors, you need to start fresh using the latest notebook as google it right now mess with the installations daily
